Scripts/script-CommonServerPython.yml

```diff
@@ -529,9 +529,16 @@ script: |-
       if isinstance(data, STRING_TYPES):
           return data
```

Comment:
what about the case if the data is NOT list, but string with special character in it?

Reply:
In that case there's no need to flatten anything, and the calling functions seems to know how to handle unicode.